components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.common/src/main/java/org/wso2/carbon/identity/api/server/userstore/common/UserStoreConstants.java

```diff
@@ -93,9 +128,30 @@ public String getDescription() {
             return description;
         }
 
+        public Response.Status getHttpStatus() {
+
+            return httpStatus;
+        }
+
         @Override
         public String toString() {
             return code + " | " + message;
         }
     }
+
+    static {
+        ERROR_CODE_MAP.put(UserStoreConfigurationConstant.ErrorCodes.XML_FILE_NOT_FOUND,
```

Comment:
whithout mapping the error codes from the code, we can convert the error codes from a property file.